### ISSUE TITLE
模型对比疑问

### ISSUE THREAD

**Issue by kaka-lz** [PlanckChang/EMAformer#3]
def _select_criterion(self):
        criterion = nn.L1Loss()
        return criterion

这里使用L1 Loss作为训练损失，为什么对比的模型还是采用        criterion = nn.MSELoss() 的结果

<img width="1428" height="419" alt="Image" src="https://github.com/user-attachments/assets/60fac4c6-0796-4b14-9271-3c8437173945" />

TSlib研究者提出了公平对比框架，大家使用MSE作为loss，
作者为什么直接截取了对比模型的结果，而不是去使用L1loss的结果对比

__________________________________________________________________________________________________
这个是L1Loss下的TQnet模型在ettm2数据集下的结果
Args in experiment:
Namespace(activation='gelu', affine=0, batch_size=256, c_out=7, channel_id=1, checkpoints='./checkpoints/', cycle=96, d_ff=2048, d_layers=1, d_model=512, data='ETTm2', data_path='ETTm2.csv', dec_in=7, dec_way='pmf', decomposition=0, des='test', devices='0,1', distil=True, do_predict=False, dropout=0.5, e_layers=2, embed='timeF', embed_type=0, enc_in=7, factor=1, fc_dropout=0.05, features='M', freq='h', gpu=0, head_dropout=0.0, individual=0, is_training=1, itr=1, kernel_size=25, label_len=0, learning_rate=0.001, loss='mse', lradj='type3', model='TQNet', model_id='ETTm2_96_96', model_type='mlp', moving_avg=25, n_heads=8, num_workers=10, output_attention=False, padding_patch='end', patch_len=16, patience=5, pct_start=0.3, pred_len=96, random_seed=2024, revin=0, rnn_type='gru', root_path='./dataset/', seg_len=48, seq_len=96, stride=8, subtract_last=0, target='OT', test_flop=False, train_epochs=30, use_amp=False, use_gpu=True, use_multi_gpu=False, use_revin=1)
Use GPU: cuda:0
start training : ETTm2_96_96_TQNet_ETTm2_ftM_sl96_pl96_cycle96_seed2024
train 34369
val 11425
test 11425
        iters: 100, epoch: 1 | loss: 0.2749556
        speed: 0.0164s/iter; left time: 64.3090s
......
Epoch: 18 cost time: 1.0413894653320312
Epoch: 18, Steps: 134 | Train Loss: 0.2321816 Vali Loss: 0.2317348 Test Loss: 0.2459926
EarlyStopping counter: 5 out of 5
Early stopping
testing : ETTm2_96_96_TQNet_ETTm2_ftM_sl96_pl96_cycle96_seed2024
test 11425
**mse:0.16863201558589935, mae:0.24576306343078613**

——————————————————————————————
这个是TQnet在MSELoss下的结果：

<img width="740" height="573" alt="Image" src="https://github.com/user-attachments/assets/3e8ba624-0b23-41d3-b55d-5c8080a0c88b" />
